walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -1,27 +1,20 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <meta  charset="UTF-8" name="viewport" content="width=device-width, initial-scale=1">
+    <title>Stanley 🐈 and Raymond!</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
 
-      <p>She is very ferocious!</p>
-      <img src="images/stanley-2.jpg"/>
+<nav>
```

Comment:
No need to have a newline between the body and nav. 
Also please fix the indentation here to be 2 spaces more than the body.